2/두정수사이의합.js

```diff
@@ -0,0 +1,11 @@
+function solution(a, b) {
+  let answer = 0;
+  let c = 0;
+  if (a > b) {
```

Comment:
```suggestion
  let answer = 0;
  let c = 0;
  
  if (a > b) {
```
1. 변수명 짓는거 신경써주세요. 변수명만 보고 어떤 변수인지 알수 있도록 짓는게 좋아요!
2. 변수 선언과 구문은 항상 줄바꿈 해주셔야 돼요.